src/Core/Utils/ContainerIntrospectionInterface.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+namespace Ra {
+namespace Core {
+namespace Utils {
+
```

Comment:
Could you add some comments ? a brief at least ?

Reply:
Question: is this introspection interface motivated (and valid) only for OpenGL bindings ?

Reply:
My comment is perhaps obsolete but this introspection is not limited to OpenGL/GPU buffers but is motivated and interesting when interfacing Radium with any API that takes (eventually shared) data buffers as input (e.g. intel embree is not GPU based but requires this to define data buffers from Radium geometry).

Reply:
ok thx